### ISSUE TITLE
Is it 3.3V compatible?

### ISSUE THREAD

**Issue by GavynDracula** [Ariamelon/Ogen#2]
Hi, I wonder whether this pcb is 3.3V compatible.

**Comment by ghostlybutterfly** [comment]
Yes, the input voltage is limited by the maximum operating voltage of the high side of the level shifters, which is 5.5V.
